dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/administrative/group/EditGroupForm.java

```diff
@@ -477,8 +478,10 @@ private void addGroupSearch(Division div, Group sourceGroup, String query, int p
         	
 			results.setSimplePagination(resultCount,firstIndex,lastIndex,prevURL, nextURL);
 		}
-		
-        Table table = results.addTable("roup-edit-search-group",groups.size() + 1, 1);
+
+// DS-3205 fix		
+//        Table table = results.addTable("roup-edit-search-group",groups.size() + 1, 1);
```

Comment:
This comment is a thoughtful touch, but it isn't necessary to document patches in the code -- git preserves the changes and their authors, and can show them when you want them.